src/nav/Nav.stories.tsx

```diff
@@ -1,5 +1,5 @@
 /* eslint-disable react/jsx-props-no-spreading */
```

Comment:
@felipesoares6  I see that we are adding type to components;
 
 Example;
 
```
type Story = StoryObj<typeof CardLink>;

export const Basic: Story = {
```

Why we don't add Story type to for example this Nav ?